README.md

```diff
@@ -13,6 +13,7 @@ You can connect to Elasticsearch through `localhost:9200` and explore Kibana via
 Tag     | Elasticsearch | Kibana
 ------- | ------------- | ------
 latest  | 8.7.1         | 8.7.1
+ssl8    | 8.10.4        | 8.10.4
```

Comment:
As I mention below, we might not need to cut a tag/branch for the security-enabled container any more but provide it by default since ver 8, as the Elastic's official movement also appears to go with the security by default.